### PR TITLE
Dynamic strikethrough for unsupported versions

### PR DESCRIPTION
**Pull request by ramonsmits**
Does not require the version script to run only for applying strikethrough styling.

image

### CODE REVIEW

**Review by andreasohlund — APPROVED** [review]

**Comment by DavidBoike** [comment]
Do you intend to also remove the markup strikethrough? Or is this only supposed to handle when the script hasn't run in awhile?

**Comment by DavidBoike** [comment]
Needs to be applied on http://localhost:5000/nservicebus/upgrades/all-versions too.

Why isn't the strikethrough action applied to the first column as well?

File: nservicebus/upgrades/supported-versions.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var now = new Date();
```
Comment (DavidBoike) [inline nservicebus/upgrades/supported-versions.js:1]: Could you please wrap this whole thing in an immediately-executed function expression (IIFE) so we don't further pollute the global namespace?

```
'use strict';
(function () {
    // All your code
} ());

**Review by DavidBoike — CHANGES_REQUESTED** [review]
To put it all in one review:

* Do the IIFE
* Add to http://localhost:5000/nservicebus/upgrades/support-policy
* Add to http://localhost:5000/nservicebus/upgrades/all-versions
* Probably apply style to first column?

**Comment by ramonsmits** [comment]
> * Probably apply style to first column?

That is a link and for that reason the color isn't applied as links us the link color.

> Do you intend to also remove the markup strikethrough?

The code isn't removing any styling. It only applies in case it detects a row has an unsupported version.

> Or is this only supposed to handle when the script hasn't run in awhile?

It could be a full replacement for the script. The strikethrough is only a visual thing. The dates in the columns are leading.

**Comment by ramonsmits** [comment]
@DavidBoike I removed the `maroon` color to only apply strike through.

**Review by DavidBoike — APPROVED** [review]

**Pull request merged**